### DISCUSSION TITLE
Стрельба из машины

### DISCUSSION THREAD

**Discussion by MATPEXA** [octothorp-team/public-dobrograd#653] [Предложения]
Думаю многие  люди знают, что из машины нельзя стрелять или как-то наносить урон. Мое предложение это добавить игрокам стрельбу из машины ( ПП и Пистолет) <---- почему именно они? Объясню позже. Стрельба из машины бы дала очень много РП ситуаций, которые дают различные исходы событий, в том же Гетто часто проводят Драйв Бай и для игроков Крим. Орг думаю это понравиться. Почему же только ПП и пистолеты? Потому что довольно таки тяжело в ИРЛ держать автомат и на полном ходу стрелять. Также к этой системе подошла бы покупка или замена стекла ( У механика можно было бы купить за 7000 руб ), либо добавить систему открытий окна, чтобы не ломать стекло, а просто стрелять без него. Также есть предложение, если автомобиль едет около 15 и больше км/ч , то в таком случае прицеливаться было бы сложнее. **Думаю это стоит того, чтобы эту систему добавили, она очень поможет игровому процессу.**

**Comment by FSBJI** [comment]
Убрали стрельбу из машины, потому что люди которые стреляли из тачки убивали своих людей сидящих в тачке

**Reply by LENFYS** [reply]
А почему убрали, это же рп?

**Comment by travka4000** [comment]
Я конечно за ведь в погоне это поможет, стреляя по колесам ведь не придется шипы использовать, хотя их и так не используют
